include/blas_meta.h

```diff
@@ -222,6 +222,18 @@ struct is_complex_std
 
 #endif
 
+class unsupported_exception : public std::runtime_error {
```

Comment:
I started with `unimplemented` as you can see in [18a7bf6](https://github.com/codeplaysoftware/portBLAS/pull/524/commits/18a7bf67de8916e2999c5e0686806dea159e5536)  , but then changed it since technically it is implemented in portBLAS but not supported for the cases I am adding it. 
Yes, `unimplemented` is closer to oneMKL spec, if you strongly feel I should change it for better consistency to oneMKL put a thumbs up here and I'll update id. (@Rbiessy )

Reply:
I don't feel that strongly! If Rbiessy doesn't add anything, I'm happy for you to resolve this comment with no changes.

Reply:
That's fair, I don't feel strongly either so happy to see this comment resolved.